### PR TITLE
Update PR status and flaky test path

### PR DESCRIPTION
**Pull request by hwang-pku**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1451,7 +1451,7 @@ https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b2783
 https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b27837aa6fec,schemacrawler-utility,us.fatehi.utility.test.html.TableCellTest.td2,ID,,,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,common,io.seata.common.util.CollectionUtilsTest.encodeMap,ID,Accepted,https://github.com/seata/seata/pull/1802,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,tm,io.seata.tm.api.transaction.TransactionInfoTest.testFieldGetSetFromJson,ID,,,
-https://github.com/Slimefun/Slimefun4,282367d6ffaf60d79f1fa357d90b01ca154c44f2,.,io.github.thebusybiscuit.slimefun4.testing.tests.settings.TestMaterialTagSetting.testAllowedValue,ID,,,
+https://github.com/Slimefun/Slimefun4,d5e4149b4f4d83dd1620d4a4cbb6d876903c851c,.,io.github.thebusybiscuit.slimefun4.api.items.settings.TestMaterialTagSetting.testAllowedValue,ID,Opened,https://github.com/Slimefun/Slimefun4/pull/3179,
```
Comment (winglam) [inline pr-data.csv:1454]: For this line, is the d5e4149b4f4d83dd1620d4a4cbb6d876903c851c sha where you detected the test or where you fixed the test?

Comment (hwang-pku) [reply pr-data.csv:1454]: I didn't fix it by I find that the test path has changed. Should I maybe add another line for the repo with the new SHA of it?

Comment (hwang-pku) [reply pr-data.csv:1455]: I changed the format according to the updated QA in the latest commit.

Comment (winglam) [reply pr-data.csv:1455]: Can you please address the warning and error in the [checks](https://github.com/TestingResearchIllinois/idoft/pull/44/checks?check_run_id=3215197870)?

WARNING: On file pr-data.csv, row 1454: 
Status MovedOrRenamed should contain a note
ERROR: On file pr-data.csv: The file is not properly ordered

As the FAQ mentions, please add a note for [MovedOrRenamed](https://github.com/TestingResearchIllinois/idoft#how-to-indicate-that-a-test-is-renamed-or-moved-to-a-different-location). As for sorting, please follow the order as [described](https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test)

Comment (hwang-pku) [reply pr-data.csv]: The warning and error are now addressed.

**Comment by winglam** [comment]
LGTM. Thanks for the changes!

**Pull request merged**